src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -929,6 +929,11 @@ protected override bool IsByRefParam(ParameterSymbol parameter)
                 return parameter.RefKind != RefKind.None;
             }
 
+            protected override bool IsByRefMethod(MethodSymbol method)
```

Comment:
`=> method.RefKind != RefKind.None;`?

Reply:
just matched the style of the methods above.

---
In reply to: [153346398](https://github.com/dotnet/roslyn/pull/23363#discussion_r153346398) [](ancestors = 153346398)